quality_of_service_demo/rclcpp/src/message_lost_listener.cpp

```diff
@@ -28,6 +28,7 @@ class MessageLostListener : public rclcpp::Node
   explicit MessageLostListener(const rclcpp::NodeOptions & options)
   : Node("MessageLostListener", options)
   {
+    // Callback that will be used when a message is received.
```

Comment:
nit: the changes to comments are unrelated to this PR and should happen in an independent PR

Reply:
Done in https://github.com/ros2/demos/pull/457/commits/b7bcc1d7f0142e508aa1b99568ba8cba734591c9.
See https://github.com/ros2/demos/pull/458.